docs/sql/ddl.txt

```diff
@@ -370,6 +373,13 @@ Creating a partitioned table is done using the :ref:`partitioned_by_clause`::
 
 For further details, please refer to :ref:`partitioned_by_clause`.
 
+.. warning::
+
+    Each partition will be sharded into the number of shards specified in the
+    `CREATE TABLE` statement (or 5 by default). This can quickly lead to a high
+    number of shards in the cluster if a column with high cardinality is used
+    for partitioning.
+
```

Comment:
@ChrisChinchilla do you know if we have some sharding-best-practice document somewhere which I could reference here?